tests/wasmbus.rs

```diff
@@ -49,9 +49,10 @@ async fn wasmbus() -> anyhow::Result<()> {
         )
         .init();
 
+    let nats_host = "localhost";
```

Comment:
redundant, this is already available from https://docs.rs/url/latest/url/struct.Url.html#method.host

Reply:
I don't think I understand. Are you suggesting we call `Url::parse` first, then extract the host with `.host_str().unwrap()`?

Reply:
Yes, for example. It's confusing to have two "sources of truth", which are likely to diverge over time as we work on this test resulting in very hard-to-debug issues

Reply:
Well, ideally we would not need the `nats_host` at all https://github.com/wasmCloud/wasmCloud/pull/432#discussion_r1289650029

Reply:
We also use it here https://github.com/wasmCloud/wasmCloud/blob/77d663d3e1fd5590177ac8003a313a3edf29ab1f/tests/wasmbus.rs#L191 and https://github.com/wasmCloud/wasmCloud/blob/77d663d3e1fd5590177ac8003a313a3edf29ab1f/tests/wasmbus.rs#L391

If you'd address https://github.com/wasmCloud/wasmCloud/pull/432#discussion_r1289650029, we'd also use it there.

If you would like to define the host name - fine, but please use a constant for that then